.github/workflows/avm.platform.assign-issue.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow for notifying and assigning issues on creation
+name: Issue Creation - notify and assign team
+
+on:
+  issues:
+    types: [opened]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch: {}
+
+jobs:
+  NotifyTeams:
+    name: Notify-Team
```

Comment:
Should this not rather refer to the owner assignment?

Reply:
You are right. That was at the time, when I thought you can assign an issue to a team. You can't. Will change it

Reply:
done